### PR TITLE
172 sort raports

### PR DESCRIPTION
**Pull request by Divinare**
Closes https://github.com/HSLdevcom/bultti/issues/172

* Check from Kirsi's comment of how this is supposed to implement

### CODE REVIEW

**Review by danieldunderfelt — APPROVED** [review]
Good, but I wonder how/why the punctuation got removed 🤔

File: src/report/reportTotals.ts [hunk 1]
```diff
@@ -23,19 +26,29 @@ export function createColumnTotalCallback(
   return reportTotalFn(rows)
 }
 
-// Column totals for the sanctionSummary report.
+// Column totals for the sanctionSummary report
```
[the remark this answers is not in the document]
Comment (Divinare) [reply src/report/reportTotals.ts:29]: Not used to use dot in comments. Only seen it being used if there are 2 sentences

[the remark this answers is not in the document]
Comment (Divinare) [reply src/report/reportTotals.ts:29]: Well, don't see nothing greatly wrong in it, just some minor harmonization. If other comments have no dot, I don't like it randomly appear in 1% of comments :D

[the remark this answers is not in the document]
Comment (danieldunderfelt) [reply src/report/reportTotals.ts:29]: Yeah but why remove it? It seems so unnecessary.

And yes I do try to punctuate properly, even if it is just a short comment. Nothing worth removing.

**Pull request merged**